### ISSUE TITLE
Usage of Elastic causes high

### ISSUE THREAD

**Issue by giancolli** [Axway-API-Management-Plus/apigateway-openlogging-elk#220]
### ELK-Solution version

4.1.0

### Elastic-Stack version

7.16.3

### API-Management version

7.7 August21 Update

### Question

Hi @cwiechmann, this issue is a follow-up of [issue #180 ](https://github.com/Axway-API-Management-Plus/apigateway-openlogging-elk/issues/180).

After your analysis we applied suggested cache configurations on the envSettings:
`CACHE_API_PATHS=/api/v3/data/vehicle,/bev/vehicle/dynamic,/restForward/me,/api/pp/v4/clients/cnrd/default/profilePicture,/ime-ws/api/v1/wallet,/api/portal/v1.3/applications,/remoteservices/rs,/svds/vehicle,/rs-360/rs,/ime-cds/api,/api/mappings,/api/vehicle,/smc2/omc,/ime-ws-txns/api,/cnrd/cloudsync,/webapi,/ime-ws-wallet/api,/ime-ws-cic/api,/otpdatadelivery/api,/msgsvc/legacypoi,/api/me,/bmwfd,/ime-ws-challenge-summary/api,/otpclearance/api,/api/store,/api/kr-map,/diag,/otpreporting/api`

Nevertheless, we still face a very high amount of Cassandra Mismatch blocking when the API Builder is running.

**Other interesting point is the following:**
Considering this configuration (EMEA)
`ADMIN_NODE_MANAGER=EMEA|https://node01.***.net:8090,EMEA|https://node04.***.net:8090`

We noticed that when the API Builder is on, ONLY node04 faces a high increase in:

- CPU Usage
![image](https://user-images.githubusercontent.com/62059999/179205355-81346203-d065-4330-9bc5-9edff8a58c19.png)

- Internet traffic received (normal amount is about 5 GB/hour, with the builder goes over 50GB hour)
![image](https://user-images.githubusercontent.com/62059999/179205385-e4c936b0-5766-40ab-9963-6f87a9367396.png)

_Note:_ API Builder was on from around 10 to around 18

Can you gently help us to:

- Understand why the API Builder impacts so hard only node04
- Find a fix to reduce the stress API Builder puts on nodes, which will probably be beneficial for the Cassandra Mismatches issue.

Thanks
Gianmario

**Comment by cwiechmann** [comment]
Is this a new environment? Can you some details about your environment. Especially when you talking about node04, etc.

**Comment by giancolli** [comment]
Hi @cwiechmann, can you gently provide your Axway business mail so that we can communicate more details on that channel?

Thanks
Gianmario

**Comment by cwiechmann** [comment]
Hi @giancolli, Please check the behavior of Logstash and if it caches API-Details properly. Please check this video if you haven't done yet to understand what to look for:
https://youtu.be/rLT84dkgGAE?t=290

Let us know, the metrics you see in Stack-Monitoring.

**Comment by lucasbuschlinger** [comment]
Hi @cwiechmann,
Logstash seems to be fine, especially after starting the second node at around 11:00

![image](https://user-images.githubusercontent.com/33268715/180431394-9abdc0f7-890c-43ea-ab5e-a5707d18cf55.png)

FYI, we also have opened a corresponding Axway Case (01384086) and a meeting is scheduled for Monday, 14:00 CEST.

**Comment by cwiechmann** [comment]
Yes, that looks perfect! And what about the APM Details for the API-Builder API requests?

**Comment by lucasbuschlinger** [comment]
Sorry, I missed that part

![image](https://user-images.githubusercontent.com/33268715/180434878-6d74c966-c4df-4346-8c6c-d0b22a2df520.png)

If I get this right, this indicated that we're not caching enough paths again?

**Comment by lucasbuschlinger** [comment]
I just checked the APM dependencies and found that the above mentioned node04 is having lots of requests:

![image](https://user-images.githubusercontent.com/33268715/180435862-6c584191-e571-4d6d-a666-5d116aae8dfb.png)

This should also explain the higher load (its basically the admin node manager of the secondary cluster) - interestingly node01 is not listed as a dependency...
